composer.json

```diff
@@ -18,8 +18,8 @@
     ],
     "require": {
         "doctrine/mongodb-odm-bundle": "~3.0",
-        "doctrine/mongodb-odm": "~1.0",
-        "sonata-project/admin-bundle": "~2.3"
+        "sonata-project/admin-bundle": "~2.3.0",
```

Comment:
~2.3 will result in fetching `dev-master` or `2.4.x-dev` which are the same.

Anyway, if not fetching `2.4.x-dev` travis tests on 3.0 result in an error because SonataAdmin still uses `Symfony\Component\Validator\ValidatorInterface;` in 2.3.x instead of `Symfony\Component\Validator\Validator\ValidatorInterface;`.

So either we target a dev version of SonataAdminBundle or we wait for 2.4.x-dev to release a stable version.